src/main/resources/openapi/usermanagement.yaml

```diff
@@ -111,7 +111,7 @@ paths:
                       - type
                     properties:
                       type:
-                        $ref: '#/components/schemas/UserAuthorisation403ErrorCode'
+                        $ref: '#/components/schemas/UserAuthorisation403107ErrorCode'
```

Comment:
Bit nik-picky but should you be blending 2 numbers to make 1, its a bit confusing?

Reply:
changed